packages/e2e/README.md

```diff
@@ -99,6 +99,27 @@ Address:    addr_test1qr0c3frkem9cqn5f73dnvqpena27k2fgqew6wct9eaka03agfwkvzr0zyq
 
 You can configure any of these five wallets in your test and use any amount of tADA you need.
 
+## Blockfrost
+
+To run the Blockfrost end-to-end tests you only need to configure two providers, AssetProvider and ChainHistoryProvider, both must be configured as Blockfrost providers and a valid Blockfrost API key must be also set, make sure that in your .env file, you have the environment variables set:
```

Comment:
@rhyslbw
I have not yet touched on the docs; they came from the original commit. I will ping you once I'm done.

Reply:
@rhyslbw I modified the doc a bit in 332cf0c5323a195b81b16566c237f2770cce741b
Only these two are available to configure since the current tests cover only three providers. One of them doesn't need to be configured because of the way it's implemented.